### ISSUE TITLE
Setting custom excelExportOptions.filename is not working for excelExport 

### ISSUE THREAD

**Issue by chitwank-asu** [ghiscoding/Angular-Slickgrid#655]
### Your Environment

| Software                | Version(s) |
| -------------------- | -------- |
| Angular                  | 11.0.4       | 
| Angular-Slickgrid   | 2.24.1        | 
| TypeScript              | 4.0.2       | 

### Steps to Reproduce

1.  set gridoptions.excelExportOptions.filename  =  'customName';;
```ts
 this.gridOptions = {
      enableAutoResize: true,
      enableExcelExport: true,
      excelExportOptions: {
        filename: 'PurchaseOrders',
        exportWithFormatter: true
      }, ....
}
```
2. Click on the default export to excel from context menu, downloaded file name still showing 'export'
3. Same thing, when we click on Export to excel from the hamburger menu,  downloaded file name still showing 'export'

### Expected Behavior

When we set custom filename in excelExportOptions, and click on Export to excel from context menu or hamburger menu, the downloaded file name should be the custom name that is being set.
### Current Behavior

1. Clicking on the default export to excel from context menu, downloaded file name still showing 'export'
2. Same thing, when we click on Export to excel from the hamburger menu,  downloaded file name still showing 'export'

### Possible Solution

It seems to be happening because there is a hardcoded file name as **export** , (**_where as it should be picking the name from exportoptions is it is set there by the user_**)in angular-sclickgrid.js file file in executeGridMenuInternalCustomCommands() method for below:
```ts
  case 'export-excel':
                    this.excelExportService.exportToExcel({
                        **filename: 'export',**
                        format: FileType.xlsx,
                    });
```
and also is there for the context menu click method too:
```ts
  // show context menu: Export to Excel
        if (gridOptions && gridOptions.enableExcelExport && contextMenu && !contextMenu.hideExportExcelCommand) {
            const commandName = 'export-excel';
            if (!originalCustomItems.find((item) => item.hasOwnProperty('command') && item.command === commandName)) {
                menuCustomItems.push({
                    iconCssClass: contextMenu.iconExportExcelCommand || 'fa fa-file-excel-o text-success',
                    title: this.extensionUtility.translateWhenEnabledAndServiceExist(`${translationPrefix}EXPORT_TO_EXCEL`, 'TEXT_EXPORT_TO_EXCEL'),
                    disabled: false,
                    command: commandName,
                    positionOrder: 52,
                    action: () => this.excelExportService.exportToExcel({
                        **filename: 'export',**
                        format: FileType.xlsx,
                    }),
                });
            }
```
### Code Sample

```ts
this.gridOptions = {
      enableAutoResize: true,
      enableExcelExport: true,
      excelExportOptions: {
        filename: 'PurchaseOrders',
        exportWithFormatter: true
      },
      datasetIdPropertyName: 'Id',
      autoResize: {
        containerId: 'demo-container',
        sidePadding: 10
      },
      enableFiltering: true,
      enableCheckboxSelector: true,
      checkboxSelector: {
        hideInFilterHeaderRow: false,
        hideInColumnTitleRow: true
      },
      enableRowSelection: true,
      pagination: {
        pageSizes: [10, 15, 20, 25, 30, 40, 50, 75, 100],
        pageSize: defaultPageSize,
        totalItems: 0
      },
      presets: {
        pagination: { pageNumber: 1, pageSize: 20 }
      },
      backendServiceApi: {
        service: new GridOdataService(),
        options: {
          enableCount: this.isCountEnabled,
          version: this.odataVersion
        } as OdataOption,
        preProcess: () => this.displaySpinner(true),
        process: (query) => this.getPurchaseOrderApiCall(query),
        postProcess: (response) => {
          this.displaySpinner(false);
          this.getPurchaseOrderCallback(response);
        }
      }
    };
  }
```

**Comment by ghiscoding** [comment]
it is indeed a bug, will look at it after the holidays, however I fixed it in my other lib in the referenced PR
If you want to help and create a PR here, that would be great, nonetheless it will be merged/released on in January ;)

**Comment by chitwank-asu** [comment]
Thank you. I will try but I am on holidays too at the moment. 
Compliments of the Season and Happy New Year to you :)

**Comment by ghiscoding** [comment]
This is now fixed and released in the new version `2.25.0`, see [Release Note](https://github.com/ghiscoding/Angular-Slickgrid/releases/tag/v2.25.0).

Cheers ⭐

**Comment by chitwank-asu** [comment]
Thank you so much!
